src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics.CodeAnalysis;
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG.md.https://dependabot.com should work
```powershell
retry of URL failed with error: The SSL connection could not be established, see inner exception.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```